sections/summary.md

```diff
@@ -0,0 +1,13 @@
+## Summary
+
+### Normative Design Rules
```

Comment:
Dit lijkt nu wat dubbel op. Kunnen we de volgende structuur doen?

Een hoofdstuk genaamd "Normative Design Rules". Hierin is sectie 1 "Summary" met daarbij deze tekst en de lijst met regels. Daarna een sectie 2 "Design Rules".

Dit alles vermindert de hoeveelheid aparte secties en maakt het allemaal wat compacter.